chapter6

fgazzelloni opened the pull request.

fgazzelloni committed:
chapter6

jonthegeek reviewed (changes requested):
Sorry, I didn't have alerts on for this repo, evidently!

I can see that this one is going to fail; you library(HistData), but you didn't add it to the description. Please do so (with a `Remotes: OWNER/REPO` entry as well if it's from GitHub, which I think it is but I'm not sure).

Do you actually need/use all of those Rdata files in data? That's a lot, but we can keep them there if they're useful.

Thanks!

fgazzelloni committed:
added HistData pkg

fgazzelloni committed:
adding tidyverse

jonthegeek committed:
tidvyerse in imports, not depends

And tidy the DESCRIPTION while I'm at it.

jonthegeek reviewed (dismissed):
Works for me if it builds!

jonthegeek committed:
Add lsr.

jonthegeek reviewed (dismissed):
It builds on my machine now, so I'm hopeful that it will work!

fgazzelloni committed:
updating with tvthemes

fgazzelloni committed:
updating description

Merge branch 'ch6' of https://github.com/Fgazzelloni/bookclub-lsr into ch6

# Conflicts:
#	DESCRIPTION

jonthegeek reviewed (approved):
Hope that does it!
